scripts/test_release.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+OS_LIST=(
+    centos:7
+    centos:8
+    fedora:28
+    ubuntu:16.04
+    ubuntu:18.04
+)
+
+FAILED=()
+RET_VALUE=0
+
+# We'll use this simple tokenized Dockerfile.
+# https://serverfault.com/a/72511
+IFS='' read -r -d '' TOKENIZED <<"EOF"
+FROM {{OS}}
+
```

Comment:
If you included an `ARG channel` option you could specify different channels to this script. In `update.sh` you would use `$channel` This would let us run this test with stable/nightly/beta release channels.

Reply:
Done.